### PR TITLE
Add Laravel 9 support

### PR DESCRIPTION
**Pull request by SanderMuller**
Add Laravel 9 support
Drop Laravel 6 and 7 support
Drop PHP 7.2 and 7.3 support
Bump version to v2.0.0 because of the dropped support for older Laravel and PHP versions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,12 +27,12 @@ Simply add the following line to your ```composer.json``` and run ```composer up
...
 Or use composer to add it with the following command
 ```bash
-composer require "hihaho/laravel-encryptable-trait:^v1.3"
+composer require hihaho/laravel-encryptable-trait
 ```
 
 ## Requirements
-- illuminate/encryption ^6.0, ^7.0 or ^8.0
-- PHP 7.2, 7.3, 7.4 or 8.0
+- illuminate/encryption ^8.75 or ^9.0
```
Comment (RobertBoes) [inline README.md:34]:
```suggestion
- illuminate/encryption ^8.0 or ^9.0
```

File: composer.json [hunk 2]
```diff
@@ -22,13 +22,13 @@
     }
   ],
   "require": {
-    "php": "^7.2|^8.0",
-    "illuminate/encryption": "^6.0|^7.0|^8.0"
+    "php": "^7.4|^8.0",
+    "illuminate/encryption": "^8.75|^9.0"
```
Comment (RobertBoes) [inline composer.json:26]:
```suggestion
    "illuminate/encryption": "^8.0|^9.0"
```

File: composer.json [hunk 3]
```diff
@@ -22,13 +22,13 @@
...
   },
   "require-dev": {
     "mockery/mockery": "^1.3.3",
-    "orchestra/testbench": "^4.0|^5.0|^6.0",
-    "phpunit/phpunit": "^7.5|^8.0"
+    "orchestra/testbench": "^6.0|^7.0",
+    "phpunit/phpunit": "^8.5.23|^9.0"
```
Comment (RobertBoes) [inline composer.json:31]:
```suggestion
    "phpunit/phpunit": "^8.5|^9.0"
```

**Review by RobertBoes — APPROVED** [review]
Looks good to me.
Just one question, can we change the version requirements to be a bit less specific? So instead of targeting Laravel `^8.75` we can target `^8.0`, the same we do with Laravel 9

File: composer.json (hunk 3 above)
Comment (SanderMuller) [reply composer.json:31]: This 8.5.23 fixed https://github.com/hihaho/laravel-encryptable-trait/runs/7711346579?check_suite_focus=true, see
https://stackoverflow.com/a/70974793

File: README.md (hunk 1 above)
Comment (SanderMuller) [reply README.md:34]: PHP8.1 does not work for earlier versions of `illuminate/encryption` unfortunately, tested this by increasing the version step by step and running `composer update --prefer-lowest` until the tests passed

**Comment by snake-py** [comment]
When this will be merged, it is currently blocking me.

**Comment by SanderMuller** [comment]
> When this will be merged, it is currently blocking me.

Merged, and release has been tagged: https://github.com/hihaho/laravel-encryptable-trait/releases/tag/v2.0.0

**Pull request merged**